yajsapi/rest/activity.ts

```diff
@@ -344,8 +344,16 @@ class Batch implements AsyncIterable<Result> {
     let last_idx = 0;
     while (last_idx < this._size) {
       let any_new: boolean = false;
-      let exe_list = await this._activity.results(this._batch_id);
-      let results: yaa.ExeScriptCommandResult[] = exe_list;
+      let results: yaa.ExeScriptCommandResult[] = []
+      try {
+        results = await this._activity.results(this._batch_id);
+      } catch (error) {
+        if (error.response && error.response.status && error.response.status == 408) {
```

Comment:
Afair, it can be done with `?` optional chaining in a more readable way, but not necessarily important. can remain as it is.
`if( error?.response?.status == 408 )`

Reply:
Available in Node.js 14, we are supporting Node.js 12.